packages/world/src/modules/init/implementations/StoreRegistrationSystem.sol

```diff
@@ -25,7 +26,7 @@ import { WorldRegistrationSystem } from "./WorldRegistrationSystem.sol";
  * @author MUD (https://mud.dev) by Lattice (https://lattice.xyz)
  * @dev This contract provides functionality for the registration of store-related resources within the World framework.
  */
-contract StoreRegistrationSystem is System, IWorldErrors, LimitedCallContext {
+abstract contract StoreRegistrationSystem is System, IWorldErrors, IStoreRegistration, LimitedCallContext {
```

Comment:
wdyt of always inheriting from interfaces first? not really an issue in this case but feels cleaner to me in the sense that interfaces go before implementations (that might implement some of the interface functions).

Reply:
I don't feel strongly. Does it impact anything on the compiler side?

Reply:
in this case it doesn't, but if one of the interfaces declares one of the functions that is present in the other contracts it would not compile unless you put the interface first, so I feel following the convention I mentioned makes sense:

<img width="758" alt="Screenshot 2025-02-10 at 9 30 17 AM" src="https://github.com/user-attachments/assets/f1d37d37-c485-4da9-82d7-03c83668eeff" />

Reply:
looks like we're fairly inconsistent with it so maybe we can do a clean up pass separately?

<img width="545" alt="image" src="https://github.com/user-attachments/assets/3dd9bd0d-87a6-45e9-bc70-a7bc6a951a05" />